[TW-1198] Doc Runner response/request body is not available for team members (#4883)
postmanlabs/postman-docs: src/pages/docs/collections/running-collections/intro-to-collection-runs.md

    * **Data** - A [data file](/docs/collections/running-collections/working-with-data-files/) for the collection run.
    * **Persist responses for a session** - Log the response headers and bodies so you can review them after running the collection. For large collections, persisting responses may affect performance.

>         > Request and response details are persisted locally during your current Postman session and aren't saved permanently. Signing out of Postman, signing into another device with the same account, or refreshing your browser will end your session and remove the logged data.

Reviewer: ```suggestion
        > During your current Postman session, request and response details are persisted locally and not saved permanently. Your session will end and your logged data will be removed when you sign out of Postman, sign in to another device with the same account, or refresh your browser.
```

        >

>         > Response and request details are available for the person who started the collection run. Other team members can't view details for collection runs that you start.

Reviewer: ```suggestion
        > Only the person who started the collection run can view response and request details. 
```
